app/vep/vep_resources.py

```diff
@@ -88,3 +88,40 @@ async def vep_status(request: Request, submission_id: str):
     except Exception as e:
         logging.debug(e)
         return response_error_handler(result={"status": 500})
+
+@router.get("/download/{submission_id}", name="download_results")
+async def download_results(request: Request, submission_id: str):
+    try:
+        workflow_status_response = await get_workflow_status(submission_id)
+        workflow_status = workflow_status_response.dict()
+        # To use out file it will require changes in nextflow and get_workflow_status endpoint
+        results_file = workflow_status['outfile']
```

Comment:
I understand that this the result of a previous PR and is out of scope in this PR; but reading the code, I realised that backend is using the word "status" here in two senses. One sense is the whole information object about a workflow — this is the return value of `workflow_status_response.dict()`. The other sense is the actual status of the submission, which you are accessing via `workflow_status['status']`. I personally would have found this confusing; and would have probably called the `workflow_status_response` something like `workflow_info_response` (and would have made similar name changes in the model). I'm leaving it entirely up to you to consider whether this ambiguity of meaning is something that is bothering you as well.